docs/pages/dev-guide/developer-guide.md

```diff
@@ -1002,7 +1002,17 @@ plans:
         tasks: [sidecar]
 ```
 
-To initiate this plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`. Its progress can be monitored like any other plan: by issuing GET requests against the `/v1/plans/side-car-example` endpoint.
+Note that the command definition for the sidecar task includes an environment variable, `PLAN_PARAMETER`, that is not defined elsewhere in the service definition. This parameter can be supplied by the user at the time that the plan is initiated, with that parameter being propagated to the environment of every task launched by the plan. To initiate the plan, execute an HTTP POST request against the endpoint `/v1/plans/sidecar-example/start`, with the header `Content-Type: application/json` set and with a JSON body consisting of environment variable name/value pairs as in the following:
+
+```json
+{
+  "PLAN_PARAMETER": "sidecar"
+}
+```
+
+Note that when no parameters are specified, the POST request's body should consist of an empty JSON object (`{}`). Default values can be modeled using standard Bash syntax; in the above case, a desired default value for `PLAN_PARAMETER` of `sidecar` could be specified by changing the task's command string to `echo ${PLAN_PARAMETER:-sidecar} >> output`.
+
```

Comment:
Suggest:

When no parameters are specified, the body of the POST request must be an empty JSON object (`{}`). Model default values with standard Bash syntax. In the above case, you can declare the default value of `PLAN_PARAMETER` to be `sidecar` by changing the task's command string to `echo ${PLAN_PARAMETER:-sidecar} >> output`.